packages/overlay/src/active-overlay.css

```diff
@@ -30,48 +30,58 @@ governing permissions and limitations under the License.
 
     to {
         opacity: 0;
-        transform: var(--animation-transform);
+        transform: var(--sp-overlay-from);
     }
 }
 
 :host {
     z-index: 2;
     position: absolute;
-    display: none;
-    opacity: 0;
-    top: -999em;
-    left: -999em;
-    animation-duration: var(--spectrum-global-animation-duration-100);
-    animation-timing-function: ease-in-out;
+    display: inline-block;
+    pointer-events: none;
 }
 
 :host([state]) {
-    display: block;
+    display: inline-block;
 }
 
-:host([state='visible']) {
+:host([state][position]) {
+    display: inline-block;
```

Comment:
Is it needed to set `display: inline-block;` on `:host` three different ways?

Reply:
Wow, good catch. Historical cruft. Fixed